### PR TITLE
fix: handle change for source.max set to sliderMaxPosition

### PR DESCRIPTION
**Pull request by marjonlynch**
# Pull Request

### General

- [x] I have included a change request file using `$ yarn change`
- [ ] I have added tests for my changes.
- [ ] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

### CODE REVIEW

**Review by chrisdholt — APPROVED** [review]
Look at that! Thanks for the fix and catching this @marjonlynch!

**Review by robarbms — APPROVED** [review]

**Review by EisenbergEffect — CHANGES_REQUESTED** [review]
Can you change the `type` and `dependentChangeType` to `prerelease`?

File: change/@microsoft-fast-foundation-191270ff-a804-4a51-85b3-ecd5192d664b.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```
Comment (EisenbergEffect) [inline change/@microsoft-fast-foundation-191270ff-a804-4a51-85b3-ecd5192d664b.json:2]: Need to change `type` to "prerelease" as well.

**Review by EisenbergEffect — CHANGES_REQUESTED** [review]

File: change/@microsoft-fast-foundation-191270ff-a804-4a51-85b3-ecd5192d664b.json (hunk 1 above)
Comment (chrisdholt) [reply change/@microsoft-fast-foundation-191270ff-a804-4a51-85b3-ecd5192d664b.json:2]:
```suggestion
  "type": "prerelease",
```

**Review by EisenbergEffect — APPROVED** [review]

**Pull request merged**